### PR TITLE
Update for loadouts and arsenals

### PR DESCRIPTION
**Pull request by Kartsaa**
Changed loadouts/arsenals to more up-to-date with current modpack we have.
Also changed Project BLUFOR based factions to work with new update dropping in a few days.
Plus a couple of my own created faction files.

As an extra, edited custom.sqf to provide a more in-depth info on how to write weapons and attachments.

### CODE REVIEW

**Review by OverkillGuy — COMMENTED** [review]
Am I correct that these are exclusively changes of "asset referenced as part of faction definition"?

If so, having no way to confirm these are "good values" to use, I'm a useless reviewer =)
I assume you got lots of testing for all that already?

**Comment by Kartsaa** [comment]
> Am I correct that these are exclusively changes of "asset referenced as part of faction definition"?
> 
> If so, having no way to confirm these are "good values" to use, I'm a useless reviewer =) I assume you got lots of testing for all that already?

Factions folder has the files that are referenced/loadouts loaded from when you choose a loadout template in the arsenal, and the arsenals are the contents of the arsenal itself.

So, basically good values are "Does hit country use this and this weapon and gear".

Testing done for most of them, only thing that would be of concern at this point are just typos.

**Comment by OverkillGuy** [comment]
Good shit, massive congrats to undertaking this!

Typos would mostly crash that file/loadout, right? So as long as we load each once, and it looks vaguely right...?

**Comment by Kartsaa** [comment]
Yeah, typos cause the loadout either not to work, or it just wont load that weapon/magazine/attachment/gear.
Load each one to test them. (Though have to wait for Project BLUFOR update before being able to test most of them)

File: Zeus_yymmdd_Template.Stratis/factions/custom.sqf [hunk 1]
```diff
@@ -5,6 +5,13 @@
 // Make sure the ammo fits the weapon.
 // Give best allowed vest, backpack, and helmet that players are allowed (e.g. armored vest instead of chest rig).
 // Give tracer/mixed ammo if possible.
+// Each item needs to be within quotes ""
+// If there is multiple items, meaning attachments and mags, all of the are insdie [] as well
+// Each attachment needs to be withing quotes, and then after each attatchment there needs to be a comma after the quotes
+// You can have maximum of 4 attachments, but you can have zero as well. In that case just have [""] in where the attachment should be
+// Example; ["weapon", "mag", [""]];
+// Each line needs to end in a ;
+// Basic info for a weapon line: ["weapon_name", "magazine_name", ["attachment1","attachment2","attachmen3","attachment4"], "secondary_ammunition_if_any"];
```
Comment (Gehock) [inline Zeus_yymmdd_Template.Stratis/factions/custom.sqf]:
```suggestion
// Each item needs to be within double quotes: ""
// If there are multiple items, meaning attachments and mags, all of the are inside [] as well
// Each attachment needs to be within quotes, and then after each attachment there needs to be a comma after the quotes
// You can have maximum of 4 attachments, but you can have zero as well. In that case just have [""] in where the attachment should be
// Example; ["weapon", "mag", [""]];
// Each line needs to end in a ;
// Basic info for a weapon line: ["weapon_name", "magazine_name", ["attachment1","attachment2","attachment3","attachment4"], "secondary_ammunition_if_any"];
```

**Pull request merged**